### PR TITLE
Label issues with mobile when android or ios are present

### PR DESCRIPTION
**Pull request by natke**

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ ep:TensorRT: '/(\btensor\s*rt\b|\btrt\b)/i'
 ep:tvm: '/\btvm\b/i'
 ep:VitisAI: '/\bvitis(?:ai)?\b/i'
 platform:jetson: '/\bjetson\b/i'
-platform:mobile: '/(\bobj(?:ective)?-?c\b|\bnnapi\b|\bcore-?ml\b|\bmobile\b)/i'
+platform:mobile: '/(\bobj(?:ective)?-?c\b|\bnnapi\b|\bcore-?ml\b|\bmobile\b|\bandroid\b|\bios\b)/i'
```
Comment (skottmckay) [inline .github/labeler.yml:14]: should this also have 'xamarin' and 'maui'/'MAUI'?

**Comment by faxu** [comment]
If you are updating this again based on the suggestions above, could you also update to add a "platform:web" label for issues including "webgl" and "wasm"?

File: .github/labeler.yml (hunk 1 above)
Comment (natke) [reply .github/labeler.yml]: Added

**Comment by natke** [comment]
> If you are updating this again based on the suggestions above, could you also update to add a "platform:web" label for issues including "webgl" and "wasm"?

Added

**Review by skottmckay — APPROVED** [review]

**Review by faxu — APPROVED** [review]

**Pull request merged**